content/contribute/index.adoc

```diff
@@ -0,0 +1,24 @@
+---
+title: "Contribute"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+menu:
+  main:
+    weight: 30
+---
+
+## Reporting issues and feature requests
+
+The Kiali team uses link:http://issues.jboss.org/browse/KIALI[JIRA at JBoss.org] as its preferred tracker. Please use it to report issues and create new feature requests (select issue type 'Bug' for bugs and 'Story' for enhancement requests).
```

Comment:
[Here](https://github.com/kiali/kiali.io/pull/22/files#diff-501311b690b3b0fe5f1d56d224f2f61eR22) I'm writing:
`In general, we use the GitHub tracker for issues raised by the community, and JIRA for issues raised by the core engineering team. Both are publicly accessible.`
Which one is correct?
See also what @pilhuhn wrote, but it hasn't been merged: https://github.com/kiali/kiali/pull/369/files

Reply:
@jotak I think this PR is to move the existing md into adoc.
After that is done we can change the wording, but this is technical task.

Reply:
Ah, correct, I didn't see the removed "contribute/index.md".